### ISSUE TITLE
version number is wrong in CLI, from prior release

### ISSUE THREAD

**Issue by mmomjian** [rfjakob/cshatag#29]
Thanks for the new release!

In both the prior release and this one, the CLI version number doesn't match. I am running v2.2.0

```
user@host:~$ cshatag
cshatag v2.1.0-12-g03d9680
Usage: cshatag [OPTIONS] FILE [FILE2 ...]
Options:
  -dry-run
        don't make any changes
  -fix
        fix the stored sha256 on corrupt files
  -q    quiet: don't print <ok> files
  -qq
        quiet²: Only print <corrupt> files and errors
  -recursive
        Recursively descend into subdirectories. Symbolic links are not followed.
  -remove
        Remove any previously stored extended attributes.
user@host:~$
```

**Comment by rfjakob** [comment]
Shoot, thanks. Makefile bug. I have fixed the bug and released v2.2.1.

**Comment by mmomjian** [comment]
Cool, thanks! Love the program
